x-pack/plugins/security/server/authentication/api_keys/api_keys.test.ts

```diff
@@ -407,6 +408,43 @@ describe('API Keys', () => {
     });
   });
 
+  describe('validate()', () => {
+    it('returns false when security feature is disabled', async () => {
+      mockLicense.isEnabled.mockReturnValue(false);
+      const result = await apiKeys.validate({
+        id: '123',
+        api_key: 'abc123',
+      });
+      expect(result).toBeFalsy();
```

Comment:
__nit:__ (just to be precise)
```suggestion
      expect(result).toEqual(false);
```